src/Microsoft.Health.Fhir.Shared.Client/Microsoft.Health.Fhir.Shared.Client.projitems

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)FhirClient.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FhirClientExtensions.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)FhirException.cs" />
+    <Compile Include="..\Microsoft.Health.Fhir.Shared.Client\FhirClientException.cs" />
```

Comment:
nit: any specific reason why we are not using $(MSBuildThisFileDirectory) in this line?

Reply:
Because VS removed it when it did the name change, I'll update it.